### ISSUE TITLE
How to deinstall

### ISSUE THREAD

**Issue by Humpelman** [vikt0rm/dbus-goecharger#10]
Hey! 
Thanks for your work!
Sadly my Victron MP2 GX is overloaded and its even not possible to do any firmware update anymore. Thats why I have to deinstall severall shelly and the go-e from the venus OS.
How can I deinstall your code?
BR

**Comment by tm-107** [comment]
There should be a uninstall.sh in the program-directory ...

**Comment by vikt0rm** [comment]
Correct, please use the uninstall.sh script to deinstall this extension.
